style.css

```diff
@@ -478,7 +475,178 @@ textarea:focus {
   font-size: 32px;
 }
 
+.pop-up {
+  display: none;
+}
+
+.pop-up-menu {
+  display: block;
+  top: 0;
+  left: 0;
+  right: 0;
+  padding: 10px;
+  background-color: #fff;
+  z-index: 10;
+  color: #3c3a39;
+  justify-content: center;
+  bottom: 0;
+  border: 1px solid #dfe1e6;
+  position: fixed;
+  overflow: hidden;
+}
+
+/* .multi-x {
+  display: flex;
+  flex-direction: row;
+  width: 259px;
+  height: 82px;
+  font-size: 32px;
+} */
```

Comment:
- [OPTIONAL] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments.